transfer/src/main/java/org/kinecosystem/transfer/receiver/view/AccountInfoActivityBase.java

```diff
@@ -66,10 +70,21 @@ public void close() {
     }
 
     @Override
-    public void onTransactionInfoReceived(String senderAppName, String memo, String receiverAppId , String senderAppId) {
+    public void onTransactionInfoReceived(String senderAppName, String memo, String receiverAppId, String senderAppId) {
         TextView title = findViewById(R.id.transfer_title);
-        final CharSequence destinationApp = getApplicationInfo().loadLabel(getPackageManager());
-        title.setText(getString(R.string.receiver_activity_message, senderAppName, destinationApp));
+        title.setText(getString(R.string.receiver_activity_message, senderAppName, receiverAppName));
+    }
+
+    @Override
+    public void showErrorDialog(AccountInfoError dataError, String... params) {
+        updateAlertTitle(dataError, params);
+        AccountInfoErrorDialog dialog = new AccountInfoErrorDialog(this, dataError);
+        dialog.show();
+    }
+
+    @Override
+    public void lunchMainActivity() {
```

Comment:
Not sure I am going to like having a 'MainActivity' for lunch 🍜 😉

Reply:
its because i am hungry all the time